index.js

```diff
@@ -6,7 +6,7 @@ const getExtensions = extensions => extensions.length > 1 ? `{${extensions.join(
 
 const getPath = (filepath, cwd) => {
 	const pth = filepath[0] === '!' ? filepath.slice(1) : filepath;
-	return path.isAbsolute(pth) ? pth : path.join(cwd, pth);
+	return path.isAbsolute(pth) ? pth : path.posix.join(cwd, pth);
```

Comment:
While debugging https://github.com/sindresorhus/globby/pull/126, I found out using `path.posix.join` in this line is excess.

A returned path by `getPath()` is used only for check whether there is a directory (via `path-type`). Thus, it must join by OS-specific separator for returning correct path.